experiments/spotify/1_no_code_consume_source/README.md

```diff
@@ -0,0 +1,36 @@
+# Consume a ready source without Python code
+We can load a source from the command line. Imagine this:
+
+```
+pip install python-dlt
+pip install python-dlt-contrib
+
+dlt init
+
+vi .dlt/secrets
+
+dlt source info dlt.contrib.sources.spotify
```

Comment:
I believe you mention this below, but can this just be aliased as `spotify` or at least `sources.spotify`